### PR TITLE
build: add sb topic auth rule listener

### PR DESCRIPTION
**Pull request by Mech0z**
## Description

We also need a listener to make the subscription useful

## References

* #0000

### CODE REVIEW

File: build/infrastructure/main/sb-charges.tf [hunk 1]
```diff
@@ -126,6 +126,16 @@ resource "azurerm_servicebus_subscription" "sbs-link-command-received" {
   max_delivery_count  = 1
 }
 
+module "sbtar_command_rejected_listener" {
```
Comment (prtandrup) [inline build/infrastructure/main/sb-charges.tf:129]:
```suggestion
module "sbtar_link_command_received_listener" {
```

Comment (HenrikSommer) [inline build/infrastructure/main/sb-charges.tf:129]: I assume this should have been named "sbtar_command_received_listener"?

Comment (lasrinnil) [inline build/infrastructure/main/sb-charges.tf:129]: 2 modules with the same name is this correct? (I don't really know)

**Review by lasrinnil — APPROVED** [review]

File: build/infrastructure/main/sb-charges.tf (hunk 1 above)
Comment (Mech0z) [reply build/infrastructure/main/sb-charges.tf]: Fixed in 8b35a70e0aa3aae464d4afb6b2eec1bbab21d9ac

Comment (Mech0z) [reply build/infrastructure/main/sb-charges.tf]: Fixed in 8b35a70e0aa3aae464d4afb6b2eec1bbab21d9ac

**Review by prtandrup — APPROVED** [review]

**Pull request merged**